RecipeMatch/API/APIManager.m

```diff
@@ -0,0 +1,48 @@
+//
+//  APIManager.m
+//  RecipeMatch
+//
+//  Created by ellabuysse on 7/8/22.
+//
+
+#import "APIManager.h"
+#import "LikedRecipe.h"
+@import Parse;
+
+@implementation APIManager
+
++ (instancetype)shared {
+    static APIManager *sharedManager = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sharedManager = [[self alloc] init];
+    });
+    return sharedManager;
+}
+
+- (void)getRecipes:(void(^)(NSArray *recipes, NSError *error))completion{
+    //Do any additional setup after loading the view.
+    NSURL *url = [NSURL URLWithString:@"https://api.edamam.com/api/recipes/v2?type=public&q=apple&app_id=00fb2355&app_key=1020f34ec9260531e5ad653a90e2d111"];
```

Comment:
For now it's ok for it to be formatted like that, but in the future, you can move the api params (url/appid/appkey) to be constants